src/pages/EditMonitor.vue

```diff
@@ -899,6 +902,28 @@
                                     </div>
                                 </template>
                             </template>
+
+                            <!-- TLS Options -->
+                            <template v-if="monitor.type === 'port-tls'">
+                                <h2 class="mt-5 mb-2">{{ $t("TLS Options") }}</h2>
+
+                                <!-- Use STARTTLS -->
+                                <div class="my-3 form-check">
+                                    <input id="start-tls" v-model="monitor.startTls" class="form-check-input" type="checkbox">
+                                    <label class="form-check-label" for="start-tls">
+                                        {{ $t("Use STARTTLS") }}
+                                    </label>
+                                </div>
+
+                                <!-- Request -->
+                                <div class="my-3">
+                                    <label for="request" class="form-label">{{ $t("Request") }}</label>
+                                    <textarea id="request" v-model="monitor.request" class="form-control"></textarea>
```

Comment:
can you include a placeholder of an example request?

Reply:
See my comment in the previous conversation. I think that should take care of this.